docs/user/itools/list-dynamic-simulation-models.md

```diff
@@ -0,0 +1,29 @@
+# iTools list-dynamic-simulation-models
+
+The `list-dynamic-simulation-models` command list all models used by the [time domain](../../simulation/dynamic/index.md) simulation for a given provider.
+
+## Usage
+```
+usage: itools [OPTIONS] list-dynamic-simulation-models [--dynamic-models]
+       [--event-models] [--help]
+
+Available options are:
+    --config-name <CONFIG_NAME>   Override configuration file name
+
+Available arguments are:
+    --dynamic-models   display implemented dynamic models
+    --event-models     display implemented event models
+    --help             display the help and quit
+
+```
+
+### Optional options
+
+`--dynamic-models`  
+This option allows to display dynamic models only. (by default all models will be displayed)
```

Comment:
Could you factorize the "By default, all models are displayed" by writing something like: "By default, all models are displayed but you can use one of the following options to display specific models"